examples/models/toolbars.py

```diff
@@ -1,3 +1,13 @@
+''' This example shows multiple ways to place the toolbar and the x and y axes 
+with respect to the plot
+
+.. bokeh-example-metadata::
+    :apis: bokeh.layouts.column, bokeh.layouts.row, bokeh.plotting.figure, bokeh.plotting.show
+    :refs: :ref:`ug_interaction_positioning-the-toolbar`
+    :keywords: tools, toolbar, axes, layout
+
+'''
+
 import numpy as np
```

Comment:
consistency with other examples
```suggestion
'''
import numpy as np
```